microfrontends/SignDocuments/src/Component/PdfRequestFiles.js

```diff
@@ -573,7 +573,7 @@ function PdfRequestFiles() {
           //if pos.isMobile false -- placeholder saved from desktop view then handle position in mobile view divided by scale
           if (pos.isMobile) {
             const x = pos.xPosition * (pos.scale / scale);
-            return x * scale + 50;
```

Comment:
Can we generalize the similar logical code in utils.js ?
So we can make change in one file.

Reply:
ok